roles/rsyslog_client/tasks/client.yml

```diff
@@ -0,0 +1,157 @@
+---
+- name: Get information on local CA certificate
+  ansible.builtin.stat:
+    path: "{{ rsyslog_local_dir }}/{{ rsyslog_ca_cert_file }}"
+    checksum_algorithm: sha1
+  register: local_rsyslog_ca_cert
+  delegate_to: localhost
+  connection: local
+
+- name: If local CA certificate exists, deploy it
```

Comment:
No need to put ```if``` in the name of a task; the ```when``` statement is pretty descriptive and when the role is running on the commandline the ```skipped``` indicates it was not necessary to perform this task.